R/summarise_scores.R

```diff
@@ -96,6 +111,16 @@ summarise_scores <- function(scores,
     by <- forecast_unit
   }
 
+  # if across is provided, remove from by
+  if (!is.null(across)) {
```

Comment:
We might want to add a check / a warning / error if there are any columns in across that aren't columns in the data.

Reply:
There is currently an error if none of the columns in across are in by but can generalise to be if any aren't

Reply:
Extended and tested here:  354fff33cb30255121522922d9396cc6645be4a5